style.css

```diff
@@ -79,4 +81,224 @@ body {
   background: #1a2236;
 }
 
-/* ------------------------------------------------------- */
+/* -------------------Work Section CSS-------------------------------- */
+
+.work {
+  position: absolute;
+  width: 320px;
+  height: 2134px;
+  left: 0;
+  top: 672px;
+}
```

Comment:
- [ ] Kindly display your work section with grid according to the requirement. 👍